src/spaced/spaced/scenes/home.cpp

```diff
@@ -15,6 +17,9 @@ Home::Home(bave::App& app, Services const& services) : Scene(app, services, "Hom
 	auto const& resources = services.get<Resources>();
 	auto const tasks = std::array{
 		util::create_font_atlas_task(resources.main_font, {TextHeight{100}, TextHeight{60}}),
+		std::function<void()>{[] { std::this_thread::sleep_for(1s); }},
```

Comment:
what's this doing?

Reply:
Oops this needs to be reverted, nice catch! It's basically test code to "see" the progress bar update when loading the home scene. Since there's only one task we never get to see it move.